src/app/server/Server.h

```diff
@@ -139,6 +141,8 @@ class Server
     ServerTransportMgr mTransports;
     SessionManager mSessions;
     CASEServer mCASEServer;
+    CASEClientPoolDelegate * mCASEClientPool         = nullptr;
```

Comment:
Maybe we can directly store the object rather then injecting the pointer with `SetCASEClientPool`. Since OTA is a must-have feature for most devices we need this object anyway.